openassessment/fileupload/backends/s3.py

```diff
@@ -0,0 +1,63 @@
+import boto
+import logging
+from django.conf import settings
+
+logger = logging.getLogger("openassessment.fileupload.api")
+
+from .base import BaseBackend
+from ..exceptions import FileUploadInternalError
+
+
+class Backend(BaseBackend):
+
+    def get_upload_url(self, key, content_type):
+        bucket_name, key_name = self._retrieve_parameters(key)
+        try:
+            conn = _connect_to_s3()
+            upload_url = conn.generate_url(
+                expires_in=self.UPLOAD_URL_TIMEOUT,
+                method='PUT',
+                bucket=bucket_name,
+                key=key_name,
+                headers={'Content-Length': '5242880', 'Content-Type': content_type}
```

Comment:
I realize that this magic number was likely copied from the old implementation. But - can a comment be added as to why it's this value?